source/ways-of-working/troubleshooting/index.html.md.erb

```diff
@@ -0,0 +1,97 @@
+---
+title: Troubleshooting issues
+last_reviewed_on: 2021-03-25
+review_in: 6 months
+weight: 99
+---
+
+# <%= current_page.data.title %>
+
+## Flux/ Gitops
```

Comment:
I think this should have a space:

![image](https://user-images.githubusercontent.com/21194782/112478221-af7aa680-8d6b-11eb-8ee3-46535aa97c61.png)


```suggestion
## Flux / Gitops
```